esp/esp/program/modules/handlers/teacherclassregmodule.py

```diff
@@ -190,6 +188,7 @@ def teacherDesc(self):
             'class_full': """Teachers teaching a completely full class""",
             'class_nearly_full': """Teachers teaching a nearly-full class (>%d%% of capacity)""" % (100 * capacity_factor),
             'taught_before': """Teachers who have taught for a previous program""",
+            'taught_before_and_now': """Teachers who have submitted a class and who have taught for a previous program""",
```

Comment:
Shouldn't this just be a combo list search?

Reply:
yes. but i dont know how to add something to the dashboard without doing it that way-- let me know if there's a way.